Miscompilation in version 0.35.19: transformCondition is not defined

Under some conditions babel errors out with this error:
```
transformCondition is not defined
npm ERR!     at /project/node_modules/babel-plugin-jsx-dom-expressions/index.js:2655:84
```

Looking at the latest published code for this package, it seems there is a miscompilation:
https://www.npmjs.com/package/babel-plugin-jsx-dom-expressions/v/0.35.19?activeTab=code

on line 431: `function transformCondition$1(path, inline, deep)`
on line 2656: `transformCondition(attribute.get("value").get("expression")`

Looks to be the same in 0.36.0-beta.6

--- a/packages/babel-plugin-jsx-dom-expressions/src/universal/element.js
+++ b/packages/babel-plugin-jsx-dom-expressions/src/universal/element.js
@@ -8,7 +8,8 @@ import {
   getConfig,
   getRendererConfig,
   convertJSXIdentifier,
-  canNativeSpread
+  canNativeSpread,
+  transformCondition
 } from "../shared/utils";
 import { transformNode } from "../shared/transform";
 
@@ -329,10 +330,11 @@ function processSpreads(path, attributes, { elem, hasChildren, wrapConditionals 
       const dynamic =
         isContainer && isDynamic(attribute.get("value").get("expression"), { checkMember: true });
       if (dynamic) {
-        const id = convertJSXIdentifier(node.name)
+        const id = convertJSXIdentifier(node.name);
         let expr =
           wrapConditionals &&
-          (t.isLogicalExpression(node.value.expression) || t.isConditionalExpression(node.value.expression))
+          (t.isLogicalExpression(node.value.expression) ||
+            t.isConditionalExpression(node.value.expression))
             ? transformCondition(attribute.get("value").get("expression"), true)
             : t.arrowFunctionExpression([], node.value.expression);
         runningObject.push(

